### DISCUSSION TITLE
Nap schedule

### DISCUSSION THREAD

**Discussion by gullygossner** [babybuddy/babybuddy#391] [Ideas]
Not sure how tough it would be to implement but a nap schedule option would be nice. Maybe the user could input the number of naps wanted throughout the day with another input for wake windows between said naps. 

Could do something like end last nap-wake window 1=next scheduled nap, end last nap-wakw window 2=next scheduled nap, etc.

**Comment by cdubz** [comment]
#362 is related I think, and something I'm open to.

**Reply by gullygossner** [reply]
Yes you are correct it is related, apologize for skimming over that.
